ppr-ui/src/components/collateral/composables/useVehicle.ts

```diff
@@ -1,5 +1,5 @@
 import { reactive, toRefs, computed } from '@vue/composition-api'
-import { VehicleTypes } from '@/resources'
+import { VehicleTypes, VehicleTypesNoMH } from '@/resources'
```

Comment:
What is VehicleTypesNoMH used for?

Reply:
A Repairer's Lien registration cannot have MH in the list.

Reply:
I don't see this resource actually used anywhere :)

Reply:
EditCollateral.vue line 33

Reply:
Gotcha! Sorry I missed that